src/dialects/mssql/index.js

```diff
@@ -85,6 +85,15 @@ assign(Client_MSSQL.prototype, {
     })
   },
 
+  prepBindings(bindings) {
+    return map(bindings, (value) => {
+    if ( typeof value === 'number' && value >= 2147483647 || value <= -2147483648){
```

Comment:
Why use these numbers? This is to prevent loss of integer precision right?

The actual valid range is [[Number.MIN_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MIN_SAFE_INTEGER), [Number.MAX_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)].